stories/corporate-data.js

```diff
@@ -562,5 +562,36 @@ export const corporateMembers = [
 				position: 'observer'
 			}
 		]
+	},
+	{
+		entity: {
+			type: 'corporate',
+			companyType: 'Company Limited by Shares (LTD)',
+			companyName: 'Selfkey Org',
+			email: 'info@selfkeyorg'
+		},
+		attributes: [
+			{
+				title: 'incorporation',
+				name: 'incorporation',
+				type: {
+					content: {
+						$id: 'http://platform.selfkey.org/schema/attribute/legal-jurisdiction.json',
```

Comment:
We should really start tracking the attributes that we are explicitly using somewhere in constants, separate issue though